### PR TITLE
Symfony 2.7 compatibility

### PR DESCRIPTION
**Pull request by chives**
Merge after merge'ing and tagging of https://github.com/fsi-open/doctrine-extensions/pull/59

### CODE REVIEW

**Comment by chives** [comment]
@fsi-open

File: spec/FSi/Bundle/DoctrineExtensionsBundle/Twig/Extension/AssetsSpec.php [hunk 1]
```diff
@@ -13,8 +13,13 @@
 use FSi\DoctrineExtensions\Uploadable\File;
 use PhpSpec\ObjectBehavior;
 use Prophecy\Argument;
-use Symfony\Bundle\TwigBundle\Extension\AssetsExtension;
-use Symfony\Component\Routing\RequestContext;
+
+class AssetsExtension
+{
+    public function getAssetUrl($path)
```
Comment (szymach) [inline spec/FSi/Bundle/DoctrineExtensionsBundle/Twig/Extension/AssetsSpec.php]: is this a placeholder?

Comment (chives) [inline spec/FSi/Bundle/DoctrineExtensionsBundle/Twig/Extension/AssetsSpec.php]: This method is implemented in https://github.com/symfony/TwigBridge/blob/master/Extension/AssetExtension.php (>=2.7) or https://github.com/symfony/TwigBundle/blob/2.6/Extension/AssetsExtension.php (<2.7). This class allows compatibility with both versions.

**Comment by szymach** [comment]
:+1:

**Comment by rn0** [comment]
:+1:

**Comment by JarekW** [comment]
:+1:

**Pull request merged**